src/Microsoft.VisualStudio.AppDesigner/PropPageDesigner/PropertyPageSerializationService_Store.vb

```diff
@@ -459,7 +459,6 @@ Namespace Microsoft.VisualStudio.Editors.PropPageDesigner
 
                     If TypeOf Member Is PropertyDescriptor Then
                         Dim Prop As PropertyDescriptor = DirectCast(Member, PropertyDescriptor)
-                        Debug.Assert(Prop.PropertyType.IsSerializable)
```

Comment:
[Here](http://index/?left=outline&rightProject=Microsoft.VisualStudio.ProjectSystem.Implementation&file=PropertyPages%5CPageEnumPropertyBase.cs&line=53) you can see that the base type implementing the interface overrides the property that gives the Property Type. My initial assumption was that CPS did not want to expose their implementation types and just stick with interfaces. @lifeng @jviau Could you explain why the property was overriden?